style-guide/check-space.py

```diff
@@ -0,0 +1,15 @@
+# -*- coding: utf-8 -*-
```

Comment:
Does this exclude inline literal blocks, which might have dot notation? Like: `object.method()` or `example.com`?